core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -413,10 +413,13 @@ private[spark] class SparkSubmitArguments(args: Seq[String], env: Map[String, St
       case ("--help" | "-h") :: tail =>
         printUsageAndExit(0)
 
-      case ("--verbose" | "-v") :: tail =>
+      case ("--verbose") :: tail =>
         verbose = true
         parse(tail)
 
+      case ("--version" | "-v") :: tail =>
+        SparkSubmit.printVersionAndExit()
```

Comment:
IMHO, we may just need to print the version information as simple as possible,  just like what the option `help` dose.